frontend/src/components/ShootDetails/GardenctlCommands.vue

```diff
@@ -125,7 +222,21 @@ export default {
     },
     toggle (index) {
       Vue.set(this.expansionPanel, index, !this.expansionPanel[index])
+    },
+    invokeCommandString (command) {
+      switch (this.shell) {
+        case 'powershell':
+          return `& ${command} | Invoke-Expression`
+        case 'fish':
+          return `eval (${command})`
+        default:
+          return `eval $(${command})`
+      }
     }
+  },
+  async mounted () {
+    await this.initializeClusterIdentity()
+    await this.refreshGardenctlOptions()
```

Comment:
This method invokes `SET_GARDENCTL_OPTIONS` in the store. This method updates the local storage with the current value. This means, that when the user first opens an arbitrary  shoot details page, the local storage gets initialized / updated with the values, even if the user never actually modified the settings. While this is not a real problem, I just wanted to point out that if the admin somehow misses to set the config values and users did open a detail shoot page once, the commands are not updated to the admin provided values (which the admin may provide shortly after) even if the user did not actively change them.